gordlea/fireboard2mqtt
# release version 3.x

This is a complete rewrite in rust. It should fix many longstanding issues around the fireboard drive, as well as be much more memory and cpu efficient.

Tested with: 
* Fireboard2
* Fireboard Spark
* Fireboard Pro
* Yoder YS640S

Resolves #11 
Fixes #32 
Fixes #33
Fixes #38 

### How to beta test this release

1. Go to home-assistant addons, and then uninstall fireboard2mqtt.
2. Go to the addon store page, and remove "gordlea's Home Assistant add-on repository".
3. Add the repository back in with the following url: `https://github.com/gordlea/home-assistant-addons#beta`
4. Reinstall the fireboard2mqtt addon, it should be version 3.0.0-beta.x.
5. Configure the addon with your fireboard credentials
